servers/src/test/java/tachyon/worker/block/TieredBlockStoreTestUtils.java

```diff
@@ -77,72 +90,164 @@ public static TachyonConf newTachyonConf(int[] tierLevel, StorageLevelAlias[] ti
+   * @param tierAlias Alias of this tier.
+   * @param tierPath Path of this tier. When `baseDir` is specified, the actual test tierPath
+   *        turns into `baseDir/tierPath`.
+   * @param tierCapacity Capacity of this tier.
+   * @param workerDataFolder When specified it sets up the tachyon.worker.data.folder property.
+   * @return The created TachyonConf.
+   * @throws Exception When error happens during creating temporary folder.
+   */
+  public static void setupTachyonConfWithSingleTier(String baseDir, int tierLevel,
+      StorageLevelAlias tierAlias, String[] tierPath, long[] tierCapacity, String
+      workerDataFolder) throws Exception {
+    if (baseDir != null) {
+      tierPath = createDirHierarchy(baseDir, tierPath);
     }
-    return tachyonConf;
+    if (workerDataFolder != null) {
+      sTachyonConf.set(Constants.WORKER_DATA_FOLDER, workerDataFolder);
+    }
+    sTachyonConf.set(Constants.WORKER_MAX_TIERED_STORAGE_LEVEL, String.valueOf(1));
+    setupTachyonConfTier(tierLevel, tierAlias, tierPath, tierCapacity);
   }
 
   /**
+<<<<<<< HEAD
+   * Sets up a specific tier's {@link TachyonConf} for a {@link TieredBlockStore}.
+=======
    * Creates a BlockMetadataManager with {@link #defaultTachyonConf}.
+>>>>>>> upstream/master
+   *
+   * @param level Level of the tier.
+   * @param tierAlias Alias of the tier.
+   * @param tierPath Absolute path of the tier.
+   * @param tierCapacity Capacity of the tier
+   */
+  private static void setupTachyonConfTier(int level, StorageLevelAlias tierAlias,
+      String[] tierPath, long[] tierCapacity) {
+    Preconditions.checkNotNull(tierPath);
+    Preconditions.checkNotNull(tierCapacity);
+    Preconditions.checkArgument(tierPath.length == tierCapacity.length,
+        String.format("tierPath and tierCapacity should have the same length"));
+
+    sTachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_ALIAS_FORMAT, level),
+        tierAlias.toString());
+
+    String tierPathString = StringUtils.join(tierPath, ",");
+    sTachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_PATH_FORMAT, level),
+        tierPathString);
+
+    String tierCapacityString = StringUtils.join(ArrayUtils.toObject(tierCapacity), ",");
+    sTachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_QUOTA_FORMAT, level),
+        tierCapacityString);
+  }
+
+  /**
+   * Joins baseDir with all the paths listed in the array and then create the new generated path.
    *
-   * @param baseDir the directory path as prefix for paths of directories in the tiered storage. The
+   * @param baseDir The directory path as prefix for all the paths in the array 'dirs'.
+   * @param dirs 2-D array of directory paths.
+   * @return New joined and created paths array.
+   * @throws Exception When error happens during creating temporary folder.
+   */
+  private static String[][] createTierHierarchy(String baseDir, final String[][] dirs)
```

Comment:
Actually at first these two APIs did share the same name as overloading, while Fanbin suggests me to distinguish them... You may get this from early comments. So... What should I do now? In my mind, both is reasonable for this case.